examples/device/net_lwip_webserver/src/usb_descriptors.c

```diff
@@ -159,13 +157,12 @@ uint16_t const* tud_descriptor_string_cb(uint8_t index, uint16_t langid)
 
   unsigned int chr_count = 0;
 
-  if (STR_LANGID == index)
+  if (STRID_LANGID == index)
   {
     memcpy(&_desc_str[1], string_desc_arr[0], 2);
```

Comment:
Could "string_desc_arr[0]" become "string_desc_arr[STRID_LANGID]"?

Reply:
ah right, I did missed that